<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketDialog</class>
 <widget class="QDialog" name="TicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ticket Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="ticketGroup">
     <property name="title">
      <string>Ticket Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
         <widget class="QLabel" name="sourceLabel">
           <property name="text">
             <string>From:</string>
           </property>
         </widget>
       </item>
       <item row="0" column="1">
         <widget class="QLabel" name="sourceValue"/>
       </item>
       <item row="1" column="0">
         <widget class="QLabel" name="destLabel">
           <property name="text">
             <string>To:</string>
           </property>
         </widget>
       </item>
       <item row="1" column="1">
         <widget class="QLabel" name="destValue"/>
       </item>
       <item row="2" column="0">
         <widget class="QLabel" name="dateLabel">
           <property name="text">
             <string>Date:</string>
           </property>
         </widget>
       </item>
       <item row="2" column="1">
         <widget class="QLabel" name="dateValue"/>
       </item>
       <item row="3" column="0">
         <widget class="QLabel" name="seatLabel">
           <property name="text">
             <string>Seat:</string>
           </property>
         </widget>
       </item>
       <item row="3" column="1">
         <widget class="QLabel" name="seatValue"/>
       </item>
       <item row="4" column="0">
         <widget class="QLabel" name="distanceLabel">
           <property name="text">
             <string>Distance:</string>
           </property>
         </widget>
       </item>
       <item row="4" column="1">
         <widget class="QLabel" name="distanceValue"/>
       </item>
       <item row="5" column="0">
         <widget class="QLabel" name="fareLabel">
           <property name="text">
             <string>Fare:</string>
           </property>
         </widget>
       </item>
       <item row="5" column="1">
         <widget class="QLabel" name="fareValue"/>
       </item>
       <item row="6" column="0">
         <widget class="QLabel" name="discountLabel">
           <property name="text">
             <string>Discount:</string>
           </property>
         </widget>
       </item>
       <item row="6" column="1">
         <widget class="QLabel" name="discountValue"/>
       </item>
       <item row="7" column="0">
         <widget class="QLabel" name="totalFareLabel">
           <property name="text">
             <string>Total Fare:</string>
           </property>
         </widget>
       </item>
       <item row="7" column="1">
         <widget class="QLabel" name="totalFareValue"/>
       </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
       <widget class="QPushButton" name="printButton">
         <property name="text">
           <string>Print</string>
         </property>
       </widget>
     </item>
     <item>
       <widget class="QPushButton" name="saveButton">
         <property name="text">
           <string>Save</string>
         </property>
       </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>